### PR TITLE
Display total balance if recv_only is enabled

### PR DESCRIPTION
**Pull request by alfonsoromanz**
Hey. Thanks for putting this tool together.

I want to use the tool to query my LND balance from another application. I noticed that the balance returned is based on a daily spending limit rather than the actual balance of the node. I propose a change to display the real node balance when the tool is set up in `recv_only` mode. This would provide a more accurate representation of the node's balance in scenarios where receiving is the primary function.

Also, it might not make much sense to display the default daily spending limit as the balance if it's set to receive only. What do you think?

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -521,14 +521,25 @@ async fn handle_nwc_params(
                 }
             }
             RequestParams::GetBalance => {
-                let tracker = tracker.lock().await.sum_payments();
-                let remaining_msats = config.daily_limit * 1_000 - tracker;
-                info!("Current balance: {remaining_msats}msats");
+                let balance: u64 = if config.recv_only() {
+                    // fetch local balance from lnd
+                    let channel_balance_request = ChannelBalanceRequest {};
+                    let channel_balance_response = lnd.channel_balance(channel_balance_request).await?.into_inner();
```
Comment (benthecarman) [inline src/main.rs]:
```suggestion
                    let channel_balance_response = lnd.channel_balance(ChannelBalanceRequest {}).await?.into_inner();
```

**Review by benthecarman — CHANGES_REQUESTED** [review]
need to fix formatting too, otherwise this makes sense to me

File: src/main.rs (hunk 1 above)
Comment (alfonsoromanz) [reply src/main.rs]: Fixed. Thanks

**Comment by alfonsoromanz** [comment]
Addressed feedback in 51d1b2a51ea353a3e205302677565e4b780ef377. Thanks

**Pull request merged**